From 927267b9157d7fb095c262cf1b2bf5d7686b1dea Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Armin=20Krezovi=C4=87?= <krezovic.armin@gmail.com>
Date: Fri, 30 Sep 2016 14:11:14 +0200
Subject: [PATCH 25/61] libweston: Drop requirement of setting
 mm_width/mm_height in backends
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

They were required for transitional phase in order not to
break previous weston_output_init(). Now, they can even
be initialized on enable, or left with defaults if backend
doesn't support them.

Reviewed-by: Pekka Paalanen <pekka.paalanen@collabora.co.uk>
Signed-off-by: Armin KrezoviÄ‡ <krezovic.armin@gmail.com>
---
 libweston/compositor.c | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/libweston/compositor.c b/libweston/compositor.c
index 2f83313..d552e18 100644
--- a/libweston/compositor.c
+++ b/libweston/compositor.c
@@ -4514,9 +4514,6 @@ weston_output_enable(struct weston_output *output)
 	if (!wl_list_empty(&c->output_list))
 		x = iterator->x + iterator->width;
 
-	/* Make sure the width and height are configured */
-	assert(output->mm_width && output->mm_height);
-
 	/* Make sure the scale is set up */
 	assert(output->scale);
 
-- 
2.7.4

